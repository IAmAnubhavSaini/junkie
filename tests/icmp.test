#!/bin/sh

NAME="ICMP"
PCAPS="pcap/icmp/protocol-unreachable.pcap pcap/icmp/destination-unreachable.pcap"
filter() {
	grep '^ICMP' | cut -d' ' -f 2-
}
OUTPUT="icmp.out"

. "$srcdir/dump_test"
