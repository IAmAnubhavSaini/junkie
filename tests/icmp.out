-- pcap/icmp/protocol-unreachable.pcap --
head_len=28, payload=0, type=ProtoUnreachable, err=protocol=224, src=172.25.3.66:0, dst=172.25.3.255:0, id=-1
